ufls_detection/ufls_detection.R

```diff
@@ -41,6 +41,7 @@ calc_sampled_time_per_circuit <- function(ts_data, start_time, end_time){
            where ts >= test_point and
                  test_point >= min_join_time"
   ts_data <- sqldf(query)
+  ts_data <- mutate(ts_data, ts = as.POSIXct(ts, origin='1970-01-01'))
```

Comment:
The sqldf query converted the timestamps into unix time, and specifically for Luceo data it then couldn't compare it to other timestamps because the origin wasn't set.

Reply:
Jack to look into database generation to make sure DB is consistent in terms of data type.